### ISSUE TITLE
Cursor Hangs on Windows Client

### ISSUE THREAD

**Issue by JonLim** [synergy/synergy#4628]
Server: OSX 10.10, Mac Mini (Late 2014)
Client: Windows 8.1

After getting SSL to work (throwing up SSL issues until I registered the client on a fresh install) the cursor would constantly hang on the Windows side of things. Seems like the Synergy client itself would hang, causing the appearance of cursor hanging.

Let me know if I can provide additional info!

**Comment by speaker** [comment]
Does this issue persist with 1.7.3+?

**Comment by JonLim** [comment]
Yes. Is there a way to test whether the issues occur because of possible connectivity issues or whether the app is just having trouble synchronizing or something else?

**Comment by speaker** [comment]
@JonLim , sorry for the delay getting back. I haven't encountered this issue myself, however, there are a lot of complexities so that's fairly common.  Mostly we would want to look over the logging at a DEBUG level on both the server and client, but there is a decent bet that the problem is SSL issues between the server and client and drag'n'drop or clipboard problems. We've been trying to address issues in wide swings by fixing overarching problems we've found in the communications rather than spending time to recreate every scenario and fixing them individually. This is both good and bad news for issues on github because we'll end up fixing a lot of problems without realizing they were there in the first place. For example, it's very easy for the plugin to end up out of sync with the synergy client or server which can be very confusing for us as well as our users.

In addition to a number of specific fixes to improve SSL connection stability, we've changed the way the SSL plugin is delivered to coming form the installer directly and then either copied into or linked from the users space. The newest nightly builds of master are very similar to the final 1.7.4 version and can be found here: http://synergy-project.org/nightly?filter=master if you're interested in trying them out. 

If you're still having this issue, (either with the new nightly build or with current 1.7.3 stable), please turn logging to DEBUG and see if you can reproduce the problem. If so, you can use gist (https://gist.github.com/) for large logfiles, or just paste into this issue.

thanks!
